docs/ops/sort/MatrixNMS_8.md

```diff
@@ -0,0 +1,157 @@
+##  MatrixNms<a name="MatrixNms"></a> {#openvino_docs_ops_sort_MatrixNms_8}
+
+**Versioned name**: *MatrixNms-8*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: *MatrixNms* performs matrix non maximum suppression (NMS) of the boxes with predicted scores.
+
+**Detailed description**: *MatrixNms* is a multi-phase operation.
+
+First it selects candidate bounding boxes that have higher scores than `score_threshold`. Then for each class at most `nms_top_k` number of candidate boxes are selected. Score of the candidate boxes are then decayed according to Matrix NMS algorithm [Wang et al](https://arxiv.org/abs/2003.10152.pdf).This algorithm is applied independently to each class and each batch element. Boxes of `background_class` are skipped and thus eliminated during the process. Finanly, it thresholds the decayed scores with `post_threshold`, and selects at most `keep_top_k` scoring candidate boxes per batch element.
```

Comment:
Missing space character in this part of the text `2003.10152.pdf).This`.

Reply:
Not sure exactly where is the missing space. Is it between "pdf)" and "This"? That's the only difference I find compared to NonMaxSuppression spec.

Reply:
Yes, add a space before "This"